tests/components/person/test_significant_change.py

```diff
@@ -0,0 +1,11 @@
+"""Test the Person significant change platform."""
+from homeassistant.components.person.significant_change import (
+    async_check_significant_change,
+)
+
+
+async def test_significant_change():
+    """Detect Person significant changes."""
+    attrs = {}
+    assert not async_check_significant_change(None, "home", attrs, "home", attrs)
+    assert async_check_significant_change(None, "home", attrs, "not_home", attrs)
```

Comment:
It should also add tests to show attribute changes do not trigger a significant change in this case.

Reply:
Tested this by passing in a source change in the `assert not` test.